CLOUDFRONT.md

```diff
@@ -2,13 +2,11 @@
 1. Login to AWS console https://aws.amazon.com/console/
 2. Navigate to _Services -> S3_.
 3. Click _Create bucket_.
-4. Fill out bucket name, region and click _Create bucket_.
-5. Navigate to _My Security Credentials_.
-6. In the _Access keys_ section click on the _Create New Access Key_ button.
-7. Write down your bucket name, region, key and secret.
-8. Edit the bucket again.
-9. Set _Default encryption_ to _Enabled_ with _Amazon S3 master-key (SSE-S3)_ server-side encryption.
-10. Set the following as _Cross-origin resource sharing (CORS)_:
+4. Fill out the bucket name and region
+5. Ensure _Block all public access_ is ticked
+6. Enable _Server-side encryption_ with _Amazon S3 key (SSE-S3)_
+7. Click _Create bucket_.
+8. Set the following as _Cross-origin resource sharing (CORS)_:
```

Comment:
Should we set Cross-origin resource sharing and _then_ click Save changes?

Reply:
The S3 interface has changed - CORS cannot be configured until after the bucket has been created.

Reply:
Ok, makes sense